x-pack/legacy/plugins/monitoring/server/lib/cluster/get_clusters_from_request.js

```diff
@@ -76,20 +87,26 @@ export async function getClustersFromRequest(req, indexPatterns, { clusterUuid,
     const cluster = clusters[0];
 
     // add ml jobs and alerts data
-    const mlJobs = await getMlJobsForCluster(req, esIndexPattern, cluster);
+    const mlJobs = isInCodePath(codePaths, [CODE_PATH_ML])
+      ? await getMlJobsForCluster(req, esIndexPattern, cluster)
+      : [];
```

Comment:
Should this maybe be `null` instead of `[]`? Since the if condition below is checking for `null`

Reply:
Yup, nice catch. You might find more, but [as stated earlied](https://github.com/elastic/kibana/pull/42469#issuecomment-518347391), the actual implementation might be slightly rough in it's current state. I didn't want to invest too much time before getting the approval on the direction first.